cmd/epp/health.go

```diff
@@ -34,10 +34,10 @@ type healthServer struct {
 
 func (s *healthServer) Check(ctx context.Context, in *healthPb.HealthCheckRequest) (*healthPb.HealthCheckResponse, error) {
 	if !s.datastore.PoolHasSynced() {
-		s.logger.V(logutil.VERBOSE).Info("gRPC health check not serving", "service", in.Service)
+		s.logger.V(logutil.TRACE).Info("gRPC health check not serving", "service", in.Service)
```

Comment:
```suggestion
		s.logger.V(logutil.DEFAULT).Info("gRPC health check not serving", "service", in.Service)
```

Actually, this is better be default, the other one TRACE, wdyt?

Reply:
If you agree, you can just use the "commit the suggestion" button above.